examples/stage-2-shield-study/addon/bootstrap.js

```diff
@@ -126,6 +131,7 @@ async function chooseVariation() {
   return toSet;
 }
 
+// TODO marcrowo: Move to lib?
```

Comment:
regarding `chooseVariation`.  i took it OUT of the lib once, because it's side behaviour is non-obvious, unless you are looking at the code.  That said, I and kamyar found this way of debugging super useful, so we left it in the bootstrap.js.